team-builder/src/components/Users.js

```diff
@@ -2,26 +2,21 @@ import React from 'react';
 
 const Users = props => {
   console.log(props)
-
-  if(props.addUserCB){
-    return(
-      <div>
-        {props.addUserCB.map(user => {
-          return (
-            <div>
-              <h2>{user.name}</h2>
-              <p>{user.id}</p>
-              <p>{user.email}</p>
-              <p>{user.role}</p>
-            </div>
-          )
-        })}
-      </div>
-    )
-  }
-  return (
-    <p>No Users</p>
-  );
```

Comment:
removed check if addusercb exists... yeah... addusercb will never exist, because we passed in a callback for userList... woops